### PR TITLE
feat(nc): add completion

### PR DESCRIPTION
**Pull request by mamercad**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

New completion

**What is the current behavior? (You can also link to an open issue here)**

There is no completion for `nc` currently

**What is the new behavior (if this is a feature change)?**

There will be completion for `nc`

**Additional info:**

`man nc`

### CODE REVIEW

File: src/nc.ts [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+  parserDirectives: {
+    optionsMustPrecedeArguments: true,
+  },
+  options: [
+    {
+      name: "-4",
+      description: "Forces nc to use IPv4 addresses only",
+      args: {
+        isOptional: true,
+        isDangerous: false,
```
Comment (SeparateRecords) [inline src/nc.ts:15]: This line isn't necessary because `isDangerous: false` is the default

**Review by SeparateRecords — CHANGES_REQUESTED** [review]
Looks good to me! One minor change then it's ready to merge: `isDangerous: false` is in a few places, which isn't necessary since that's the default behavior. I've annotated the first one but it should be easy enough to find/delete the rest!

File: src/nc.ts (hunk 1 above)
Comment (mamercad) [reply src/nc.ts]: I'll fix this up in the rest of my open PRs as well.

**Review by SeparateRecords — APPROVED** [review]

**Comment by SeparateRecords** [comment]
Fantastic, thanks so much for this! Will review your other PRs as well

**Pull request merged**